scripts/onnx/test.sh

```diff
@@ -48,6 +48,44 @@ if [[ "$SHARD_NUMBER" == "2" ]]; then
   xdoctest torch.onnx --style=google --options="+IGNORE_WHITESPACE"
 fi
 
+if [[ "$SHARD_NUMBER" == "2" ]]; then
+  # Sanity check on torchbench w/ onnx
+  pip install pandas
+  log_folder="test/.torchbench_logs"
+  device="cpu"
```

Comment:
Do we still want `cpu` only as device or should we add a `for device in "$devices[@]}"; do`?

Reply:
cpu only since CI for us is cpu only